maven/fetch.bzl

```diff
@@ -1,47 +1,165 @@
-load(":constants.bzl", "DOWNLOAD_PREFIX")
+#
+# Description:
+#   Utilities and rules concerning the actual fetch of artifacts and sub-artifacts and metadata.
+#
+load(":artifacts.bzl", "artifacts")
+load(":globals.bzl", "DOWNLOAD_PREFIX", "PACKAGING_TYPE_FILE", "fetch_repo")
+load(":packaging_type.bzl", "packaging_type")
+load(":poms.bzl", "poms")
+load(":utils.bzl", "strings")
+load(":xml.bzl", "xml")
 
 _ARTIFACT_DOWNLOAD_BUILD_FILE_TEMPLATE = """
 package(default_visibility = ["//visibility:public"])
 exports_files([
-    "{path}",
-])
+{files}])
```

Comment:
So this is a template, and this does make the format work out, when it's populated.  This makes it show a file on each line, indented, if there is one, or if none, it shows:

```
package(default_visibility = ["//visibility:public"])
exports_files([
])
```

I format the "files" string before plugging it in, to make sure the end  build file looks reasonable. And since it's an arbitrary number of files (because it can be one pom, or a pom plus several parents), I can't know at template time how many.